eng/testing/linker/trimmingTests.targets

```diff
@@ -96,7 +96,9 @@
       <TestPublishTrimmedCommand>$(TestPublishTrimmedCommand) build /t:PublishTrimmed</TestPublishTrimmedCommand>
       <TestPublishTrimmedCommand>$(TestPublishTrimmedCommand) /nr:false</TestPublishTrimmedCommand>
       <TestPublishTrimmedCommand>$(TestPublishTrimmedCommand) /warnaserror</TestPublishTrimmedCommand>
+      <TestPublishTrimmedCommand>$(TestPublishTrimmedCommand) /bl:$([MSBuild]::NormalizePath('$(ArtifactsDir)', 'log\$(Configuration)\TrimmingTestsSubTask.binlog'))</TestPublishTrimmedCommand>
```

Comment:
It would be good to upload this `binlog` to Azure DevOps, or does that just happen automatically?

cc @joperezr

Reply:
Happens automatically; been using it to try to workout what was wrong with the packaging for the trimming tests in the very many commits of https://github.com/dotnet/runtime/pull/40877; which were from missing a few build steps to deal with the `net5.0`+`net5.0-Browser` which was choking NuGet

![image](https://user-images.githubusercontent.com/1142958/90801261-97423500-e30d-11ea-9423-dac960519367.png)

Reply:
Right, we intentionally placed the binlog for this on the right folder in order for it to get automatically uploaded to the artifacts of the build along with the rest of the build logs.

Reply:
Wasn't sure if I could just add it to the `TrimmingTests.binlog` rather than creating a new one; wasn't sure if it would be happy with use from multiple processes?

Reply:
Looking more at this - I think we need 1 binlog per test, right? The way the above works is that we call `dotnet build /t:PublishTrimmed` for each test app. Won't it be overwritting the binlog each time?

Reply:
Hmm... does look like it overwrites the last one; though it runs them sequentially so when it fails it always has the last one that caused the failure (which is why I didn't notice)